backend/src/app/logic/webhooks.py

```diff
@@ -38,7 +40,8 @@ async def process_webhook(edition: Edition, data: WebhookEvent, database: AsyncS
                 if question.options is not None:
                     for option in question.options:
                         if option.id == question.value:
-                            attributes['wants_to_be_student_coach'] = "yes" in option.text.lower()
+                            attributes['wants_to_be_student_coach'] = "yes" in option.text.lower(
+                            )
```

Comment:
Any reason why this bracket is on a new line? Looks a bit weird to me, otherwise looks good.
```suggestion
                            attributes['wants_to_be_student_coach'] = "yes" in option.text.lower()
```